src/ccextractor.c

```diff
@@ -215,27 +214,33 @@ int main(int argc, char *argv[])
 				if (ccx_options.ignore_pts_jumps)
 					ccx_common_timing_settings.disable_sync_check = 1;
 				mprint ("\rAnalyzing data in general mode\n");
-				general_loop(ctx);
+				tmp = general_loop(ctx);
+				if (!ret) ret = tmp;
 				break;
 			case CCX_SM_MCPOODLESRAW:
 				mprint ("\rAnalyzing data in McPoodle raw mode\n");
-				raw_loop(ctx);
+				tmp = raw_loop(ctx);
+				if (!ret) ret = tmp;
 				break;
 			case CCX_SM_RCWT:
 				mprint ("\rAnalyzing data in CCExtractor's binary format\n");
-				rcwt_loop(ctx);
+				tmp = rcwt_loop(ctx);
+				if (!ret) ret = tmp;
 				break;
 			case CCX_SM_MYTH:
 				mprint ("\rAnalyzing data in MythTV mode\n");
 				show_myth_banner = 1;
-				myth_loop(ctx);
+				tmp = myth_loop(ctx);
+				if (!ret) ret = tmp;
 				break;
 			case CCX_SM_MP4:
 				mprint ("\rAnalyzing data with GPAC (MP4 library)\n");
 				close_input_file(ctx); // No need to have it open. GPAC will do it for us
 				processmp4(ctx, &ctx->mp4_cfg, ctx->inputfile[ctx->current_file]);
 				if (ccx_options.print_file_reports)
 					print_file_report(ctx);
+				tmp = ctx->freport.mp4_cc_track_cnt;
+				if (!ret) ret = tmp;
```

Comment:
because of [this](https://github.com/CCExtractor/ccextractor/pull/535#issuecomment-267477706)